Bottom: c286330987d7efbc0c450fa74bcf9650a2769e1c
Top:    b2a69b143f13d43fc5937a266ea5273d1acac0f8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:58:49 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
